content/pages/sprint/2021/08early-pycht.rst

```diff
@@ -0,0 +1,79 @@
+=================================================
+Scisprint 2021 Early August: Python ZH TW Doc
+=================================================
+
+:date: 2021-06-29 15:24
+:url: sprint/2021/08early-pycht
+:save_as: sprint/2021/08early-pycht.html
+
+Scisprint 2021 Early August: Python ZH TW Doc
+=================================================
+
+This is the second sprint for for translating `Python official
+document <https://docs.python.org/3/>`__ from English to traditional Chinese
+(Taiwan; ZH TW locale).
+
+The agenda:
+
+* Improve the project scheduling. Currently, this project is calling for 
+  the coordinator, and no one is available for review. We should re-planning
+  the jobs in this project. Please feel free to share your opinions about the
+  work in the project like the thoughts about the review workflow, 
+  translation work scheduling, etc. 
+
+* Improve the review. We are facing the bottleneck of translation for
+  reviewing, and the review process is not efficient. We need to improve the review
+  process including building a review workflow or mechanism.
+
+* Improve glossary.  We have not had consistent translation of many English
+  words. Find better ways to unify how terminology is translated.
+
+* Automation and CI.  We do not have any CI at the moment, but we should.  We
+  should check for build errors for each of the PRs, use of whites spaces, and
+  daily update of source strings.
+
+* Review and discussion about previous `action items <https://hackmd.io/a-gzVVMJSjumLdpVwcZ21Q?view#Action-items>`_.
+
+We are working on the 3.9 branch of the document:
+https://github.com/python/python-docs-zh-tw/tree/3.9.
+
+Registration
+------------
+
+Please join `gather.town <https://gather.town/app/yLTe8mBDb8pogMOX/sciwork>`_
+on the day of sprint!!
+
+Date & time
+-----------
+
+14th August, Saturday, 2021, 14:00 -- 16:00
+
+Agenda
+------
+
+* **13:30 -- 14:00**: Gathering
```

Comment:
It would also be good to move the agenda, if it is updated, here.

Reply:
Thanks for the notification! I'll add some detail items here.